tcmu-runner.spec

```diff
@@ -141,6 +141,7 @@ Development header(s) for developing against libtcmu.
 %dir %{_sysconfdir}/tcmu/
 %config %{_sysconfdir}/tcmu/tcmu.conf
 %config(noreplace) %{_sysconfdir}/logrotate.d/tcmu-runner
+%config(noreplace) %{_sysconfdir}/logrotate.d/tcmu-runner.bak
```

Comment:
We should remove the tcmu-runner.bak in %install section instead of this here.

Reply:
@lxbsz I thought you added the install to fix duplicate log entry error?

https://github.com/open-iscsi/tcmu-runner/commit/c031ad575da6dd6b0ca5d8f996e2f184482d195c

Reply:
Locally I have the fixed patch with my glfs lock PR, last we hit one issue and still testing not updated yet.

Thanks.

Reply:
okay, ill just close this PR then and you can put in the fix with your PR